### PR TITLE
Add searchable field to partners and display icon on searchability status

### PR DESCRIPTION
**Pull request by suecarmol**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Added the `searchable` field to the Partner model and added a search icon that displays on collection tiles whether it is searchable via EDS or not.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
EDS only indexes a subset of our total available content. Users might want to make informed decisions about whether they should search in EDS or at the publisher's website. We can help with this by denoting in some way which resources are indexed.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T270115](https://phabricator.wikimedia.org/T270115)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Run the migrations (if you haven't)
2. Modify some collections in the `admin` interface so they are searchable and partially searchable.
3. Navigate to My Library
4. Search icons should appear on the collections that are searchable or partially searchable

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

Screen Shot 2021-09-14 at 17 53 17

Screen Shot 2021-09-14 at 17 53 35

Screen Shot 2021-09-17 at 17 41 50

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/users/templates/users/available_collection_tile.html [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {% load i18n %}
 <div class="col-xl-3 col-lg-4 col-md-6 col-sm-12">
   <div class="card collection-tile">
+    {% if available_collection.searchable == searchable %}
+      {% comment %}Translators: An icon that indicate whether a partner is searchable or not. {% endcomment %}
```
Comment (Samwalton9) [inline TWLight/users/templates/users/available_collection_tile.html:5]: Probably worth clarifying that this is hovertext for the icon - describing the text itself as an icon might be confusing, e.g.:

> Hovertext for an icon which indicates whether a collection is searchable or not.

**Review by Samwalton9 — CHANGES_REQUESTED** [review]
One change requested, and I've asked Nikki for a 2nd opinion on the design :)

File: TWLight/users/templates/users/available_collection_tile.html (hunk 1 above)
Comment (suecarmol) [reply TWLight/users/templates/users/available_collection_tile.html]: Done!

**Comment by Samwalton9** [comment]
Note the request at https://phabricator.wikimedia.org/T270115#7355033 to add a little extra to this PR before merging :)

**Comment by suecarmol** [comment]
> Note the request at https://phabricator.wikimedia.org/T270115#7355033 to add a little extra to this PR before merging :)

Done! Let me know if you need me to change anything else.

**Comment by Samwalton9** [comment]
Nitpick: Could the text not wrap under the icons in the key?

So instead of this:
image

It's like:
image

I think this makes it more readable. Otherwise this looks exactly as I sketched it!

**Comment by suecarmol** [comment]
Done!

Screen Shot 2021-09-20 at 18 16 05

**Comment by Samwalton9** [comment]
This is going to wait on https://github.com/WikipediaLibrary/TWLight/pull/795 so that users aren't confused by the key.

**Comment by suecarmol** [comment]
Merging this to the EDS-feature branch

**Pull request merged**